validate_ot_nexson.py: WARNING: No node in a tree was flagged as being the root node in "tree(id=tree1)"
